src/ape/cli/choices.py

```diff
@@ -27,7 +27,7 @@ class Alias(click.Choice):
 
     name = "alias"
 
-    def __init__(self, account_type: Optional[Type[AccountAPI]] = None):
+    def __init__(self, account_type: Optional[Type[AccountAPI]] = None):  # noqa
```

Comment:
Usually better to do a specific issue w/ noqa

Reply:
I removed it because I couldn't figure it out but just FYI Pycharm yells at you from these lines.